Gaming The Syllabus/Assets/Scripts/LevelLoader.cs

```diff
@@ -25,8 +33,6 @@ void Update()
             SceneManager.LoadScene("TestLevel");
```

Comment:
Seems like this can just be deleted at this point

Reply:
In fact anything commented out in this class can just be removed. Might as well clean it up since we're modifying it in this PR anyway